metricsbp/statsd.go

```diff
@@ -59,16 +60,27 @@ type StatsdConfig struct {
 
 	// The log level used by the reporting goroutine.
 	LogLevel log.Level
+
+	// LabelValues are the labels/tags to be attached to every metrics created
+	// from this Statsd object. For labels/tags only needed by some metrics,
+	// use Counter/Gauge/Timing.With() instead.
+	//
+	// LabelValues must be in key value pairs, or NewStatsd will panic:
+	//
+	//     LabelValues: []string{"key1", "value1", "key2", "value2"}
```

Comment:
🔕 Can we make the interface here a `map` and translate it into a list before passing to statsd?

Reply:
Good point. Done.